test/integration/functional_test.go

```diff
@@ -334,7 +334,7 @@ func validateCacheCmd(ctx context.Context, t *testing.T, profile string) {
 	}
 	t.Run("cache", func(t *testing.T) {
 		t.Run("add", func(t *testing.T) {
-			for _, img := range []string{"busybox", "busybox:1.28.4-glibc", "k8s.gcr.io/pause:latest"} {
+			for _, img := range []string{"busybox:latest", "busybox:1.28.4-glibc", "k8s.gcr.io/pause:latest"} {
```

Comment:
we need to have `latest` due to https://github.com/google/go-containerregistry/pull/404/files, which allows to have both tag and digest, hence the `latest` tag is not added automatically.